standard/expressions.md

```diff
@@ -1211,7 +1211,11 @@ The run-time processing of a function member invocation consists of the followin
   - `M` is invoked.
 - Otherwise, if the type of `E` is a value-type `V`, and `M` is declared or overridden in `V`:
   - `E` is evaluated. If this evaluation causes an exception, then no further steps are executed. For an instance constructor, this evaluation consists of allocating storage (typically from an execution stack) for the new object. In this case `E` is classified as a variable.
-  - If `E` is not classified as a variable, then a temporary local variable of `E`’s type is created and the value of `E` is assigned to that variable. `E` is then reclassified as a reference to that temporary local variable. The temporary variable is accessible as `this` within `M`, but not in any other way. Thus, only when `E` is a true variable is it possible for the caller to observe the changes that `M` makes to `this`.
+  - If `E` is not classified as a variable, or if `V` is not a readonly struct type (§16.2.2), and `E` is one of:
+    - an input parameter (§15.6.2.3), or
+    - a `readonly` field (§15.5.3), or
+    - a `readonly` reference variable or return (§9.7),
```

Comment:
Comparing with the list in https://github.com/dotnet/csharpstandard/issues/894#issuecomment-1703793806, does this set include get-only auto properties?

Reply:
It doesn't. This text shouldn't need to, as it's concerned with when parameters must be copied. A get only auto prop doesn't have parameters.

The `ref readonly` part is already covered by other rules (you can't `ref` assign a `ref readyonly` return to a writable ref.